content/en/load-testing/How-Tos/master-controller-cheatsheet.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "How to use Mastercontroller Command Options (Cheat Sheet)"
+linkTitle: "Mastercontroller Cheat Sheet"
+
+weight: 300
+type: docs
+
+description: >
+  How to make use of different mastercontroller command options. <!-- You will find the usage instructions and a list of available commands below. -->
+---
+## Mastercontroller 
+The mastercontroller is the primary tool for managing the entire load test. It calculates the load distribution, begins and ends the test and also deploys the test suite to all the agentcontrollers. At the end mastercontroller saves the test results on the file system. It can be run making use of the [interactive mode](../../manual/310-test-execution/#interactive-mode), [auto mode](../../manual/310-test-execution/#auto-mode) or [non-interactive mode (scripted commands)](../../manual/310-test-execution/#non-interactive-mode-scripted-commands).
+
+## Usage
+
+- `mastercontroller [<other options>]` runs in [interactive mode](../../manual/310-test-execution/#interactive-mode). Any command can be chosen to be executed next.
+- `mastercontroller --help` displays all the available options and a brief guide about their usage.
+- `mastercontroller -auto [<other options>]` runs a load test in [auto mode](../../manual/310-test-execution/#auto-mode) by executing all commands automatically.
+
+### Example
+
+ This will make a load test run in [auto mode](../../manual/310-test-execution/#auto-mode), in combination with [embedded mode](../../manual/310-test-execution/#embedded-mode).
+##### Windows
```

Comment:
h4 (####) should be enough?

Reply:
There wasn't a noticeable difference between h4 and h3 at the first glance, but now that I looked closely, there is a little difference. Making corrections now.